### ISSUE TITLE
[BUG] -ls23 and onwards has issue with adding iptables rules

### ISSUE THREAD

**Issue by alysak6075** [linuxserver/docker-wireguard#325]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

when there is a PostUp command iptables break:
`[#] iptables -A FORWARD -i wg0 -j ACCEPT; iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE
iptables v1.8.10 (nf_tables): Could not fetch rule set generation id: Invalid argument`

### Expected Behavior

The tunnel should be up

### Steps To Reproduce

add to Interface
`PostUp = iptables -A FORWARD -i wg0 -j ACCEPT; iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE`

### Environment

```markdown
- OS:Synology DSM 7.1
- How docker service was installed:
Container Manager
```

### CPU architecture

x86-64

### Docker creation

```bash
docker run -d \
  --name=wireguard \
  --cap-add=NET_ADMIN \
  --cap-add=SYS_MODULE \
  -e PUID=1027 \
  -e PGID=100 \
  -e TZ=America/Chicago \
  -p <PORT>:<PORT>/udp \
  -v /volume3/docker_ssd/wireguard/config:/config \
  --sysctl="net.ipv4.conf.all.src_valid_mark=1" \
  --sysctl="net.ipv4.ip_forward=1" \
  --sysctl="net.ipv6.conf.all.forwarding=1" \
  --sysctl="net.ipv6.conf.default.forwarding=1" \
  --restart unless-stopped \
  linuxserver/wireguard
```

### Container logs

```bash
[#] iptables -A FORWARD -i wg0 -j ACCEPT; iptables -t nat -A POSTROUTING -o eth0 -j MASQUERADE
iptables v1.8.10 (nf_tables): Could not fetch rule set generation id: Invalid argument
```

**Comment by def324** [comment]
I have the same issue on qnap.

uname -r
5.10.60-qnap

**Comment by xankletas** [comment]
Same issue here in a arm64

uname -a
Linux CoreELEC 5.4.210 #1 SMP PREEMPT Mon Feb 19 10:40:19 CET 2024 aarch64 GNU/Linux

I solved it, using "legacy" version --> arm64v8-1.0.20210914-legacy

**Comment by aptalca** [comment]
Looks like this here is the culprit: https://git.alpinelinux.org/aports/tree/main/iptables/iptables.post-upgrade?h=3.19-stable
tl;dr alpine 3.19 switched to nftables under the hood so kernel support for nftables is required. Most newer kernels should support it, but it seems qnap and coreelec do not.

We'll evaluate internally how best to move forward.

**Comment by def324** [comment]
My solution/workaround was to pin the version to "v1.0.20210914-ls22". That way there is no need for the "legacy" build.
